GPU/Vulkan/TextureCacheVulkan.cpp

```diff
@@ -1288,6 +1288,8 @@ void TextureCacheVulkan::SetTexture(VulkanPushBuffer *uploadBuffer) {
 			delete entry->vkTex;
 			entry->vkTex = nullptr;
 		}
+	} else {
+		// TODO: If reusing an existing texture object, we must transition it into the correct layout.
```

Comment:
Oh, oops.  I was for some reason thinking that we were `image->CreateDirect()` in this case, and just preserving the `entry->vkTex->texture_`.

We can't actually overwrite, because it might have been used previously in this frame, right?

-[Unknown]